### ISSUE TITLE
Understanding the multiple formats case

### ISSUE THREAD

**Issue by lcolladotor** [rstudio/rmarkdown#67]
Hello,

Just a quick question. If I specify multiple formats in the YAML, will the code be run once or more? I'm curious about this because I sometimes render Rmd files that take a large amount of time (and/or memory).

Alternatively, if I keep the output md file and the *_files directory from rendering one type of document, is it just a matter of then using pandoc to render it into another type of document? Or would some information be missing? Or could I use a function in `rmarkdown` to add any markup needed to the md file before using pandoc?

Basically, the first case is about knowing from the get-go that I want more than one output format but without having to process the R code more than once. The second case is when you create an output document in one format, and later one realize that you wanted it in a second format but also want to avoid re-processing the R code.

Thanks for the info!
